lib/utils.ts

```diff
@@ -140,12 +140,7 @@ export function num2hex(num: number): string {
  */
 export function getVersion(): string {
 	if (!VERSION) {
-		let revision = ''
-		try {
-			revision = execSync('git rev-parse --short HEAD').toString().trim()
-		} catch (error) {
-			// git not installed
-		}
+		const revision = execSync('command -v git || exit 0; git rev-parse --short HEAD').toString().trim()
```

Comment:
Change looks good, BTW I would wrap it in a try catch anyway, of an error happens here for any reason the app wouldn't start at all